haproxy/templates/_helpers.tpl

```diff
@@ -64,3 +64,61 @@ resolvers mydns
   hold other 3s
   resolve_retries 999
 {{- end -}}
+
```

Comment:
I think it would be good to generate the metrics part based on the number in `haproxy.redisk8s.nodeCount` / `haproxy.galerak8s.nodeCount` using `until` (https://helm.sh/docs/chart_template_guide/function_list/#until).

Reply:
Thanks will look into this

Reply:
This has been implemented.

Reply:
@anothertobi Can you check the change I made and if all is good, approve the PR?